### ISSUE TITLE
Accessibility: Tabbing & Landmarks

### ISSUE THREAD

**Issue by tiffsuen** [MikeIbberson/3merge#7]
- [ ] The page content should be reachable via tabbing (using "tabindex"). Right now, only the links are tabbable. Google devs recommend using "landmarks" which are essentially different html tags to differentiate content. 

- [ ] Aria-labels to describe page sections:

- hero section = banner
- main content = main
- nav menu = navigation
- sidebars/callouts = complementary
- copyright = contentinfo
- search bars = search

I'm looking into the WCAG to see how to meet the requirements for level A. This issue will be updated!
